modules/core/src/umatrix.cpp

```diff
@@ -386,6 +386,11 @@ void UMat::create(int d, const int* _sizes, int _type, UMatUsageFlags _usageFlag
     addref();
 }
 
+void UMat::create(const std::vector<int>& _sizes, int _type, UMatUsageFlags _usageFlags)
+{
+    create((int)_sizes.size(), _sizes.data(), _type, _usageFlags);
```

Comment:
BTW, `std::vector::data()` is C++11 feature (but it is already used in several places of OpenCV):
http://en.cppreference.com/w/cpp/container/vector/data
http://www.cplusplus.com/reference/vector/vector/data/